src/aioauth/base/database.py

```diff
@@ -81,6 +81,18 @@ async def create_authorization_code(
         await self.save_authorization_code(authorization_code)
         return authorization_code
 
+    async def get_id_token(
+        self,
+        request: Request,
+        client_id: str,
+        scope: str,
+        response_type: str,
+        redirect_uri: str,
+        nonce: str,
+    ) -> str:
+        """Returns token_id"""
```

Comment:
for the `database.py` class, I would add comments like:
`"implement for oauth2"` or `"required for openidconnect"`

Reply:
@Woile yes make sense. updated :)